data/seeds/007_messages.js

```diff
@@ -15,7 +15,7 @@ exports.seed = function (knex) {
       sent_by_profile_id: 5,
     },
     {
-      title: 'When is class?',
+      title: 'When is course?',
```

Comment:
While we are strictly sticking to 'course' in the repo, this doesn't make for great spoken word. Since this is being displayed, I'd actually leave it as 'class' here.
```suggestion
      title: 'When is class?',
```